### ISSUE TITLE
[API] API does not connect in Centos8

### ISSUE THREAD

**Issue by felipe-costa-git-hub** [ngoduykhanh/PowerDNS-Admin#734]
Hello, how are you.
Like to thank everyone involved or project was very good.

I would like to ask for help.

I cannot connect the pdns service with a web-gui via API.

Teste and error message.

Test done according to the guide power DNS.

[Export-Curl.txt](https://github.com/ngoduykhanh/PowerDNS-Admin/files/4631336/Export-Curl.txt)

Informations below.

PowerDNS installation:
Installation by the powerDNS documentation
https://doc.powerdns.com/authoritative/installation.html

and an installation guide.

https://www.linuxhelp.com/how-to-install-and-configure-powerdns

in addition to the documentation provided on the wiki.

https://github.com/ngoduykhanh/PowerDNS-Admin/wiki/Prepare-MySQL-or-MariaDB-Database-for-PowerDNS-Admin

Web GUI installation:

Docker as indicated in the guide

https://github.com/ngoduykhanh/PowerDNS-Admin

Installation problems encountered:

Server powerDNS did not allow the default local address.

I had to put my local address, 172.16.1.14.

So the service was up.

API configuration:

I did as shown in the guide.
https://doc.powerdns.com/authoritative/http-api/index.html

WEB API configuration:
![API-WEB](https://user-images.githubusercontent.com/62400103/81993074-51454900-961b-11ea-9503-1bf0a036c7da.PNG)

My pdns configuration
[pdns.conf.txt](https://github.com/ngoduykhanh/PowerDNS-Admin/files/4631312/pdns.conf.txt)

**Comment by alkanis** [comment]
Hello, might be SElinux blocking.
I have installed PowerDNS-admin on centos 8 as well (pip version), i had to tweak SElinux.

**Comment by felipe-costa-git-hub** [comment]
Hi, thanks for the reply.
How can I solve this?

**Comment by johnwc** [comment]
@alkanis to test, you can `setenforce 0`

**Comment by felipe-costa-git-hub** [comment]
Hi, thanks for replying.
I performed this procedure, however, it did not solve it.
Collecting the LOG message in the system.
It presented the following message

localhost pdns_server[2589]: [webserver] Webserver closing socket: remote (172.16.1.14) does not match the set ACL(127.0.0.1/32, ::1/128)

I resolved it as follows.

 Inside the /etc/pdns/pdns.conf file

I changed the webserver-allow-from option:
By default:
webserver-allow-from = 127.0.0.1, :: 1
I configured it in this format.
webserver-allow-from = 0.0.0.0 / 0, ::

So the API is connected, but when trying to create a domain, message 400 is given.
Looking at the LOG message.
Displays the following message:

**### pdns_server [4972]: [webserver] 9325f288-ed74-4b84-8ce0-be1d67d388fa HTTP ISE for "/ api / v1 / servers / localhost / zones": Exception: GSQLBackend unable to set metadata kind 'SOA-EDIT-API' for domain 'development-domain.com.br': Could not prepare statement: delete from domainmetadata where domain_id = (select id from domains where name =?) and domainmetadata.kind = ?: Table 'powerdnsadmin.domainmetadata' doesn't exist**

I understood that the domainmetadata table does not exist.

How should I create this table?

Just creating it in the bank without indicating the formatting of the table is enough?

If not, what is the structure of this table?
